composeml/label_times.py

```diff
@@ -321,8 +321,11 @@ def sample(self, n=None, frac=None, random_state=None):
                 sample = label.sample(n=n, random_state=random_state)
                 sample_per_label.append(sample)
 
-            labels = pd.concat(sample_per_label, axis=0, sort=False)
-            return labels
+            sample = pd.concat(sample_per_label, axis=0, sort=False)
```

Comment:
would it make more sense to do something like `sample_per_label[0].append(sample_per_label[1:])`? would that automatically maintain the metadata?

Reply:
yes, that works. will update to append samples.

Reply:
actually, the append method also doesn't propagate the metadata

Reply:
fixed by overriding `__finialize__` method to propagate metadata